systemd/fmn-sse@.service

```diff
@@ -4,11 +4,11 @@ After=network.target
 Documentation=https://github.com/fedora-infra/fmn.sse/
 
 [Service]
-ExecStart=/usr/bin/python2 /usr/lib/python2.7/site-packages/fmn/sse/sse_webserver.py
+ExecStart=/usr/bin/twistd -n -l - -y /usr/lib/share/fmn.sse/sse_server.tac
```

Comment:
Yup. ``-l, --logfile=       log to a specified file, - for stdout`` is how ``twistd`` documents it.